frontend/public/components/storage-class-form.tsx

```diff
@@ -1109,7 +1072,15 @@ export type Resources = {
 export const ConnectedStorageClassForm = connect(
   mapStateToProps,
   mapDispatchToProps,
-)(StorageClassForm_);
+)(
+  withExtensions<StorageClassPramasExtensions>({ params: isStorageClassProvisioner })(
+    StorageClassForm_,
+  ),
+);
+
+export type StorageClassPramasExtensions = {
```

Comment:
```suggestion
export type StorageClassParamsExtensions = {
```
I guess this was the intended spelling. Please fix all the occurrences.